core/os_shared.h

```diff
@@ -314,6 +314,8 @@ char *
 get_application_pid(void);
 char *
 get_application_name(void);
+bool
+get_app_args(int **app_argc_out, char ***app_argv_out);
```

Comment:
For sthg in os_shared.h we generally add an impl to the other OS too even if it's NYI

Reply:
Done.